runtime/doc/nvim_python.txt

```diff
@@ -20,16 +20,20 @@ Note: For now only the old Vim 7.3 API is supported.
 ==============================================================================
 2. Quickstart						 *nvim-python-quickstart*
 
+If you used a package manager to install Nvim, there's a good chance that
+it also provides the `neovim` Python package. If it doesn't, follow these
+steps to install the package with Python's package manager pip.
+
 To use Vim Python 2/3 plugins with Nvim, do the following:
 
 - For Python 2 plugins, make sure an interpreter for Python 2.6 or 2.7 is
   available in your `$PATH`, then install the `neovim` Python package systemwide:
   >
-    $ sudo pip install neovim
+    $ sudo pip2 install neovim
```

Comment:
IIRC this isn't the case everywhere, so perhaps a small note should be added above, maybe under quickstart?